.chalice/config.json

```diff
@@ -0,0 +1,33 @@
+{
+  "stages": {
+    "dev": {
+      "api_gateway_stage": "api",
+      "environment_variables": {
+         "ES_HOST": "search-dss-azul-commons-lx3ltgewjw5wiw2yrxftoqr7jy.us-west-2.es.amazonaws.com",
+         "ES_REGION": "us-west-2",
+         "ES_INDEX":"fb_index",
+         "HOME":"/tmp"
+      }
+    },
+    "staging": {
+      "api_gateway_stage": "api",
```

Comment:
HCA/CZI uses the pattern of setting the `api_gateway_stage` to the stage name instead of consistently using `api`. This can be helpful when working with the API Gateway, yet I'm not completely sure which is the best practice. I have found `api_gateway_stage` to the stage name to be helpful when working with the AWS API Gateway console. I would be interested in your thoughts on this.

Reply:
Makes sense to have them clearly defined, I think the api_gateway_stage changes will be obscured once you put a CNAME in front, so hopefully will be largely obscured from developers.